examples/lib/stories/parallax/parallax.dart

```diff
@@ -1,11 +1,15 @@
 import 'package:dashbook/dashbook.dart';
 import 'package:flame/game.dart';
+import 'package:flame/parallax.dart';
```

Comment:
how are we organizing wrt to flame/components vs flame/feature_x ?

Reply:
Unsure if I understood your question, I believe that both components and parallax exports the parallax stuff, this was the choice because this file don't uses components.